apps/remix-ide/src/app/files/remixDProvider.js

```diff
@@ -207,4 +208,12 @@ module.exports = class RemixDProvider extends FileProvider {
     if (!this._isReady) throw new Error('provider not ready')
     return await this._appManager.call('remixd', 'isFile', { path: unprefixedpath })
   }
+
+  isSubDirectory (parent, child) {
```

Comment:
don't you need to add `isSubDirectory` to all the provider?

Reply:
you should perhaps then add  `isSubDirectory` to `FileProvider` instead of `RemixDProvider`